pkg/chrootarchive/diff_windows.go

```diff
@@ -14,6 +15,12 @@ import (
 // contents of the layer.
 func applyLayerHandler(dest string, layer archive.Reader, decompress bool) (size int64, err error) {
 	dest = filepath.Clean(dest)
+
+	// Ensure it is a Windows-style volume path
```

Comment:
Yup, that's me not being familiar with GitHub.  I didn't realize it was interpreting my double slashes and automatically turning them into single slashes.  I meant <code>"\\\\?\\"</code>